app/forms/client_property_entry_form.rb

```diff
@@ -4,7 +4,6 @@ class ClientPropertyEntryForm < BasePropertyEntryForm
   attr_accessor :partner
 
   attribute :house_in_dispute, :boolean
-  validates :house_in_dispute, inclusion: { in: [true, false] }, allow_nil: false
```

Comment:
You're right. I was shying away from it because I think to do it properly requires a bit of a refactor that risked a can of worms, but on balance it's worth doing.